submissions/onyinyeChukwuka/HTML/index.html

```diff
@@ -1,71 +1,117 @@
+            </div>
 
-        <p> <h2> Cami's Combat Boots</h2>
-            <img src="images/combatboots1.jpg">
-            <img src="images/combatboots4.jpg">
-            <br> 
-            <br>
-            <hr>
-            <hr>
-            <br>
-            <strong> Price:</strong> &#x20A6; 25,000
-            <h4> Cami's Combat Boots</h4>
-            <hr>
-            <strong> Features</strong>
-            <br>
-            <ul>
-                <li> Round toe </li>
-                <li> Padded cuff  </li>
-                <li> Chunky sole</li>
-                <li> Lace-up fastening</li>
-            </ul>
-        </p>
-        <p>
-            <strong> DESCRIPTION</strong>
-            <br>
-            - Heavy laced leather boot especially with a wide cuff buckled above the ankle. Suitable for outdoor activities and casual outings.
-        </p>
+            <!-- FOR THE RIGHT SIDE -->
+            <div class="camidescription">
+                <h3> CAMI'S COMBAT BOOTS</h3>
+                <h4> Brand: River Island</h4>
+                <h4>Quantity: <input type="text" value="1" name="quantity " id="quantity"></h4>
+                <h4> Price: &#x20A6 15,000</h4>
+                <button type="submit" id="button"> BUY NOW</button>
+            </div>
+        </div>
+
+        <!-- FOR THE DESCRIPTION -->
+        <div class="description">
+            <h4> DESCRIPTION</h4>
+            <h3> Heavy laced leather boot especially with a wide cuff buckled above the ankle. Suitable for outdoor
+                activities and casual outings.</h3>
+        </div>
     </main>
-    <aside> 
-        <p> 
-            <strong> SIMILAR TO THIS</strong>
-            <br>
-            <ol>
-                <li> <a href="#"> Dr Martens Boots</a></li>
-                <li> <a href="#"> Jay's Trainers </a></li>
-                <li> <a href="#"> Cami's Platform boots </a></li>
-                <li> <a href="#"> Military-style boots </a></li>
-            </ol>
-        </p>
-    </aside>
-    <footer>
-        <p>© 2020 BootStraps</p>
+    <!-- for the related products section -->
+    <div class="related">
+        <h4> RELATED PRODUCTS</h4>
+        <div class="items">
+            <div class="martens">
+                <a href="#"> <img src="../html/images/drmartens.png" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4>Dr Martens Combat Boots</h4>
+                <h4> Price: &#x20A6 75,000</h4>
+            </div>
+            <div class="platforms">
+                <a href="#"> <img src="../html/images/platform.jpg" alt="Dr-martens-boots" width="399px"
+                        height="298px"></a>
+                <h4> Chain Platform Boots</h4>
+                <h4> Price: &#x20A6 35,000</h4>
+            </div>
+            <div class="military">
+                <a href="#"> <img src="../html/images/military.jpg" alt="Dr-martens-boots" width="399px"
```

Comment:
I notice that you have the same `alt` attribute values for the 3 products which are not cool enough. Use different alternate texts to describe the products.